resync/core/audit_db.py

```
@@ -76,9 +110,22 @@ async def get_entity_history(
         """Get recent audit actions."""
-        return await self._repo.get_all(limit=limit, order_by="timestamp", desc=True)
+        return await self._repo.get_all(limit=limit, offset=offset, order_by="timestamp", desc=True)
+
+    def get_record_count(self) -> int:
+        """Get total number of audit records."""
+        # Use a simplified sync wrapper since this is used in sync dashboard context
+        try:
+            asyncio.get_running_loop()
+            # If in loop, we'd need to await, but this is a shim.
+            # Returning 0 or similar is safer than crashing if called incorrectly.
+            return 0
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)